client/src/actions/index.js

```diff
@@ -41,6 +41,17 @@ async function configFetch(dispatch) {
   });
 }
 
+async function userInfoFetch(dispatch) {
+  return fetchJson("userinfo").then((response) => {
+    const userinfo = { ...response.userinfo };
```

Comment:
I suggested this change based of my assumption that the shape of the response was:
```javascript
response = {
    userinfo: {
         userinfo
    }
}
```
I'm not sure if this is actually the case.

This might be why smoke tests are failing